marda_registry/data/filetypes/biologic-mpt.yml

```diff
@@ -0,0 +1,41 @@
+---
+id: >-
+    biologic-mpt
+name: >-
+    BioLogic MPT file
+description: >-
+    A text data file created by BioLogic EC-Lab from data from BioLogic cyclers, containing
+    time- and frequency-domain data pertaining to many electrochemical techniques,
+    such as
+    voltammetry, potentiometry, coulometry and impedance spectroscopy.
```

Comment:
some strange ideas about line length from pre-commit here... not a priority but can try to fix it at the repo level after this PR